LICENSE

```diff
@@ -1,6 +1,8 @@
+Copyright for portions of project Backstage Upgrade helper are held by React Native Community, 2019 as part of React Native Upgrade Helper. All other copyright for project Backstage Upgrade helper are held by The Backstage Authors, 2021.
```

Comment:
If we're changing the owners below, this line should also be updated.

Reply:
it was